### PR TITLE
Code journal

### PR DESCRIPTION
**Pull request by Ortiz-Andrew**
https://github.com/Ortiz-Andrew/code-journal/assets/136508122/8f245986-3da8-47ea-80ee-c54b0548abbb

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+header {
+  background-color: #4c2676;
+  color: white;
+  text-align: left;
+}
+
+.column-half {
+  display: flex;
+  flex-direction: column;
+  float: left;
```
Comment (shawnkost) [inline css/styles.css]: You should not be using `float`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Besides the changes that I messaged you, this also needs to get fixed!

**Comment by Ortiz-Andrew** [comment]
Screenshot 2023-09-19 at 10 56 23 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**